vulcanai2/models/basenetwork.py

```diff
@@ -351,15 +418,15 @@ def validate(self):
                     pbar.update(10 * len(data))
                 else:
                     pbar.update(len(self.val_loader.dataset) - int(batch_idx*len(data)))
-            val_accuracy_accumulator += self.metrics.get_score(predictions, targets, metric='accuracy')
+            val_accuracy_accumulator += self.metrics.get_score(predictions, targets)
```

Comment:
why remove `metric='accuracy'`?

Reply:
because it's already the default argument in get_score

Reply:
oh okay. @rfratila was updating on it. So didn't notice if it was defaulted sorry

Reply:
I think it would be redundant to keep it but I though it could remind us that in the future you can change the metric type